File: MODUL 6/PRAK 6/PRAK604-2410817210025-Helga.c
#include <stdio.h>
#include <string.h>

int main(){
    char asli[100];
    char palsu[100];
    scanf("%99[^\n]", &asli);
    getchar();
    scanf("%99[^\n]", &palsu);
    getchar();
    int a, b;
    int x = 0, y = 0;
    a = strlen(asli);
    b = strlen(palsu);
    if(a != b){
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    else{
        for(int i = 0; i < a; i++){
            if(asli[i] == palsu[i]){
                printf("*");
                x++;
            }
            else{
                printf("#");
                y++;
            }
        }
        printf("\n* = %d", x);
        printf("\n# = %d", y);
    if(x >= y){
        printf("\nPesan Asli");
    }
    else if(x < y){
        printf("\nPesan Palsu");
    }
    }
}

File: MODUL 6/PRAK6/PRAK604-2410817210025-Helga.py
asli = input()
palsu = input()
a = len(asli)
b = len(palsu)
x = 0
y = 0
if len(asli) != len(palsu):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range(len(asli)):
        if asli[i] == palsu[i]:
            print("*", end = ' ')
            x = x + 1
        else:
            print("#", end = ' ')
            y = y + 1
    print("\n* = {}".format(x))
    print("# = {}".format(y))
    if x >= y:
        print("Pesan Asli")
    elif x < y:
        print("Pesan Palsu")